AnagramSolver.BusinessLogic/Classes/WordRepository.cs

```diff
@@ -39,6 +39,21 @@ public HashSet<WordModel> GetSpecificPage(int pageNumber)
             }
             return newList.Skip(howManySkip).Take(wordsInPage).ToHashSet();
         }
+
+        public HashSet<string> GetSpecificWords(string word)
+        {
+            var allWords = GetAllWords();
+            var newList = new HashSet<string>();
```

Comment:
Avoid using variable names like 'new'
Don't use name list when variable is in different format (in this case, Set)